### DISCUSSION TITLE
import useStore from where?

### DISCUSSION THREAD

**Discussion by vuejstime** [vuejs/pinia#663] [Help and Questions] [answered]
https://pinia.esm.dev/ssr/nuxt.html#using-the-store-outside-of-setup
i think this part of document is ambiguous i don't know import useStore from where?
nuxt composition api package?
or
it's my pinia store?
i checked nuxt-composition-api useStore doc and i think it's my pinia store but i'm not sure please fix the doc thank you.

**Comment by BenShelton** [comment, marked as answer]
Correct, it's your store. It's the same as demonstrated in https://pinia.esm.dev/core-concepts/#using-the-store

However in that example it explicitly states the import as `import { useStore } from '@/stores/counter'`, would including that line in the SSR & Nuxt docs too be helpful?

**Reply by posva** [reply]
I added the line to the docs.

Thanks for helping out with discussions @BenShelton ! I really appreciate it 💚
